operate/schema/src/main/java/io/camunda/operate/store/elasticsearch/NewElasticsearchBatchRequest.java

```diff
@@ -0,0 +1,152 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.operate.store.elasticsearch;
+
+import co.elastic.clients.elasticsearch.ElasticsearchClient;
+import co.elastic.clients.elasticsearch.core.BulkRequest;
+import co.elastic.clients.elasticsearch.core.BulkResponse;
+import co.elastic.clients.elasticsearch.core.bulk.BulkResponseItem;
+import io.camunda.operate.entities.OperateEntity;
+import io.camunda.operate.exceptions.PersistenceException;
+import io.camunda.operate.store.BatchRequest;
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Batch request that uses new style ElasticsearchClient instead of deprecated RestHighLevelClient.
+ */
+public class NewElasticsearchBatchRequest implements BatchRequest {
```

Comment:
Just leaving a note that we should consider a better name in the future. I assume we are going to remove the old ElasticsearchBatchRequest at some point?

Reply:
I really could not find the better name. Deprecated client is called `RestHighLevelClient` and Operate class that uses it is called `ElasticsearchBatchRequest`.

New client is called `ElactisearchClient` and I was not sure how to call the corresponding BatchRequest
implementation. 

Should we may be rename `ElasticsearchBatchRequest -> RestHighLevelElasticsearchBatchRequest` and name the new one `ElasticsearchBatchRequest` ?

Reply:
I think that would be a good idea, it makes it clear too that it's intended for (or uses) the RestHighLevelClient. We could even mark it deprecated as well if the plan is to eventually remove it or get away from it completely

Reply:
Full implementation here: https://github.com/camunda/camunda/pull/19111